Make some of the changes needed to the 'makeicecat' script, to allow it to run
in a snippet without network access.  After this patch is applied, some
additional changes will be made using 'substitute*'.

diff --git a/makeicecat b/makeicecat
index aa46b94..db27a86 100644
--- a/makeicecat
+++ b/makeicecat
@@ -36,75 +36,75 @@ export DEBFULLNAME="Ruben Rodriguez"
 
 DATA="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"/data
 
-mkdir output
-cd output
+# mkdir output
+# cd output
 
 ###############################################################################
 # Retrieve FF source code
 ###############################################################################
 
-rm mozilla-esr${FFMAJOR} $SOURCEDIR -rf
-
-wget -N https://ftp.mozilla.org/pub/mozilla.org/firefox/releases/${FFVERSION}esr/source/firefox-${FFVERSION}esr.source.tar.xz
-wget -N https://ftp.mozilla.org/pub/mozilla.org/firefox/releases/${FFVERSION}esr/source/firefox-${FFVERSION}esr.source.tar.xz.asc
-gpg --recv-keys --keyserver keyserver.ubuntu.com 24C6F355
-gpg --verify firefox-${FFVERSION}esr.source.tar.xz.asc
-
-echo Extracting Firefox tarball
-tar -xf firefox-${FFVERSION}esr.source.tar.xz
-
-mv firefox-${FFVERSION} $SOURCEDIR
+# rm mozilla-esr${FFMAJOR} $SOURCEDIR -rf
+# 
+# wget -N https://ftp.mozilla.org/pub/mozilla.org/firefox/releases/${FFVERSION}esr/source/firefox-${FFVERSION}esr.source.tar.xz
+# wget -N https://ftp.mozilla.org/pub/mozilla.org/firefox/releases/${FFVERSION}esr/source/firefox-${FFVERSION}esr.source.tar.xz.asc
+# gpg --recv-keys --keyserver keyserver.ubuntu.com 24C6F355
+# gpg --verify firefox-${FFVERSION}esr.source.tar.xz.asc
+# 
+# echo Extracting Firefox tarball
+# tar -xf firefox-${FFVERSION}esr.source.tar.xz
+# 
+# mv firefox-${FFVERSION} $SOURCEDIR
 
 ###############################################################################
 # Retrieve /debian from Ubuntu
 ###############################################################################
 
-rm -rf firefox.$CODENAME
-bzr branch https://code.launchpad.net/~mozillateam/firefox/firefox.$CODENAME
-cd firefox.$CODENAME
-bzr revert -r$REVISION
-echo '3.0 (native)' > debian/source/format
-
-for PATCH in ubuntu-bookmarks.patch ubuntu-ua-string-changes.patch unity-menubar.patch ubuntu-search-defaults.patch fix-make-package-tests-without-webrtc.patch revert-upstream-search-engine-changes.patch
-do
-  rm debian/patches/$PATCH
-  sed "/$PATCH/d" -i debian/patches/series
-done
-sed "/test-/d" -i debian/patches/series
-cd ..
-
-mv firefox.$CODENAME/debian $SOURCEDIR
-rm -rf firefox.$CODENAME
+# rm -rf firefox.$CODENAME
+# bzr branch https://code.launchpad.net/~mozillateam/firefox/firefox.$CODENAME
+# cd firefox.$CODENAME
+# bzr revert -r$REVISION
+# echo '3.0 (native)' > debian/source/format
+# 
+# for PATCH in ubuntu-bookmarks.patch ubuntu-ua-string-changes.patch unity-menubar.patch ubuntu-search-defaults.patch fix-make-package-tests-without-webrtc.patch revert-upstream-search-engine-changes.patch
+# do
+#   rm debian/patches/$PATCH
+#   sed "/$PATCH/d" -i debian/patches/series
+# done
+# sed "/test-/d" -i debian/patches/series
+# cd ..
+# 
+# mv firefox.$CODENAME/debian $SOURCEDIR
+# rm -rf firefox.$CODENAME
 
 ###############################################################################
 # Retrieve l10n
 ###############################################################################
 
-mkdir l10n
-cd l10n
-while read line;do
-    line=$(echo $line |cut -d' ' -f1)
-    #[ $line = "es-ES" ] || continue # To speed up testing
-    [ $line = "en-US" ] && continue
-    hg clone https://hg.mozilla.org/l10n-central/$line
-    mkdir -p $line/browser/chrome/browser/preferences
-    touch $line/browser/chrome/browser/preferences/advanced-scripts.dtd
-    rm -rf $line/.hg*
-done < ../$SOURCEDIR/browser/locales/shipped-locales
-cd ..
-
-mv l10n $SOURCEDIR
-
-hg clone http://hg.mozilla.org/l10n/compare-locales/
-cd compare-locales/
-hg checkout RELEASE_3_3_0
-cd ..
-rm compare-locales/.hg* compare-locales/.git* -rf
-mv compare-locales $SOURCEDIR/l10n
+# mkdir l10n
+# cd l10n
+# while read line;do
+#     line=$(echo $line |cut -d' ' -f1)
+#     #[ $line = "es-ES" ] || continue # To speed up testing
+#     [ $line = "en-US" ] && continue
+#     hg clone https://hg.mozilla.org/l10n-central/$line
+#     mkdir -p $line/browser/chrome/browser/preferences
+#     touch $line/browser/chrome/browser/preferences/advanced-scripts.dtd
+#     rm -rf $line/.hg*
+# done < ../$SOURCEDIR/browser/locales/shipped-locales
+# cd ..
+# 
+# mv l10n $SOURCEDIR
+# 
+# hg clone http://hg.mozilla.org/l10n/compare-locales/
+# cd compare-locales/
+# hg checkout RELEASE_3_3_0
+# cd ..
+# rm compare-locales/.hg* compare-locales/.git* -rf
+# mv compare-locales $SOURCEDIR/l10n
 
 #######################################################
 
-cd $SOURCEDIR
+# cd $SOURCEDIR
 
 #for patch in $DATA/patches/*; do
 #    echo Patching with file: $patch
@@ -720,7 +720,7 @@ debian/rules debian/control
 touch -d "yesterday" debian/control
 debian/rules debian/control
 
-echo | dch -b -D stable -v "$ICECATVERSION"  "Converted into IceCat (http://www.gnu.org/software/gnuzilla/)"
+# echo | dch -b -D stable -v "$ICECATVERSION"  "Converted into IceCat (http://www.gnu.org/software/gnuzilla/)"
 sed "1s/firefox/icecat/" -i debian/changelog
 
 touch configure js/src/configure
@@ -734,6 +734,6 @@ sed 's/777/755/;' -i toolkit/crashreporter/google-breakpad/Makefile.in
 /bin/sed 's/chmod a+w/chmod u+w/' -i ./js/src/ctypes/libffi/Makefile.in ./toolkit/crashreporter/google-breakpad/Makefile.in ./toolkit/crashreporter/google-breakpad/src/third_party/glog/Makefile.in || true
 
 
-cd ..
-echo Packaging tarball
-tar cfj icecat-$ICECATVERSION.tar.bz2 $SOURCEDIR
+# cd ..
+# echo Packaging tarball
+# tar cfj icecat-$ICECATVERSION.tar.bz2 $SOURCEDIR
